package.json

```diff
@@ -40,15 +41,27 @@
     "react-native-safe-area-context": "4.10.1",
     "react-native-screens": "3.31.1",
     "react-native-svg": "15.2.0",
+    "react-native-table-component": "^1.2.2",
```

Comment:
Is this being used?

Reply:
same re: @types/react-native-table-component

Reply:
Yes